__init__.py

```diff
@@ -10,7 +9,7 @@
 # (at your option) any later version.
 #---------------------------------------------------------------------
 
-from PyQt5.QtWidgets import QAction, QMessageBox
+from qgis.PyQt.QtWidgets import QAction, QMessageBox
```

Comment:
To be honest I prefer the existing approach. To me, `qgis.PyQt` module is just a way to ease transition from qt4 to qt5. Moreover, this makes the dev also wonder why we need this import through qgis and how is it different from 'vanilla' pyqt5.

Reply:
Ok. No problem.
You can use the squash button on github.

Reply:
Given the transition to PyQt6, @wonder-sk are you still preferring to hard-code `PyQt5` ?
I still teach to use `qgis.PyQt` as it will make the transitions to Qt6 version easier. A lot of code has been done in this way in QGIS source code to make plugins working on both versions.